manifests/vhost.pp

```diff
@@ -562,10 +562,9 @@
   # - $virtual_docroot
   # - $docroot
```

Comment:
I think these template uses comments become redundant when you explicitly pass in parameters.